Espanol

exequielpagliari opened the pull request:
Hello!

How are you?

I have translated Read.me into Spanish, it is at the bottom of it.

I just need to understand how the Links of the titles work for the references of each one and it would be fully functional.

Thanks for the software, I needed it.

exequielpagliari committed:
Terminada traducción

exequielpagliari committed:
Revisión general

exequielpagliari committed:
Mejorando Organización de Read.me con tradución

janos-ijgyarto commented:
Hey there, sorry for getting back to you this late, Github never notified me about this.

I'm more than happy to add a translation, though it may be out of date. Can we discuss it in DMs?

exequielpagliari commented:
Hello!

Thank for reply!

No problem with talk on DM.

exequielpagliari committed:
Merge branch 'janos-ijgyarto:master' into espanol

exequielpagliari committed:
Modificado el archivo README.md para adaptarse al nuevo update del proyecto.

exequielpagliari committed:
- Creation of Folder "readme"
- Extract readme.md english & spanish secction, create "english.md" and "spanish.md" with this content.

exequielpagliari committed:
- Fixed Qt version number line on "spanish.md".

exequielpagliari committed:
- Add a Index for translated .md

exequielpagliari committed:
- Repair relative link of md

janos-ijgyarto commented on readme/spanish.md:
Looks good, but you will need to adjust the markdown links to the headings, since the headings were also translated. I will highlight an example

janos-ijgyarto commented on readme/spanish.md:
```diff
@@ -0,0 +1,164 @@
+
+## Empezando
+
+Una vez que abra Hux, primero debes [importar](#importing-scenarios) un escenario desde la carpeta dividida (split folder), or [abrir](#loading-scenarios) un archivo de escenario Hux.
```
This is an example, you need to change `#importing-scenarios` to match the translated heading (this will allow users to click the link and be taken to the corresponding part of the document)

janos-ijgyarto commented on readme.md:
Since the Readme is now in separate files, the one at the root could be trimmed back to just the short summary, followed by links to the specific translations

exequielpagliari committed:
Fix: Arreglados algunos links

exequielpagliari committed:
Fix: Fix all links.

exequielpagliari commented:
Ey, @janos-ijgyarto, i repair the spanish links on Spanish.md

janos-ijgyarto commented on readme/spanish.md:
```diff
@@ -0,0 +1,164 @@
+
+## Empezando
+
+Una vez que abra Hux, primero debes [importar](#importando-escenarios) un escenario desde la carpeta dividida (split folder), or [abrir](#loading-scenarios) un archivo de escenario Hux.
```
The `abrir` is still using the english link

janos-ijgyarto commented on readme/spanish.md:
```diff
@@ -0,0 +1,164 @@
+
+### Editor de Terminal
+
+Cuando hace click en una terminal en el [Navegador de Escenarios](#navegador-de-escenarios), se abre una ventana del [Editor de Terminal](#terminal-editor) para este terminal.
```
`Editor de Terminal` also needs to be updated

exequielpagliari committed:
Fix: line 36 & 65 link
